README.md

```diff
@@ -1,109 +1,56 @@
 
-A reactive Java interface for the official [Discord](https://discordapp.com/) API, written in Java 8. Discord4J v3 is designed using [reactive principles](https://www.reactivemanifesto.org/)
-using the [Reactor framework](https://projectreactor.io/) and built over [Netty](https://netty.io/) in order to create an efficient, fast and completely non-blocking API.
+A reactive Java wrapper for the REST and Gateway components of the official [Discord Bot API](https://discordapp.com/developers/docs/intro).
 
-For a quick overview on the benefits of reactive programming/Reactor as well as a quick tutorial on Reactor click here (TODO: add link!).
+Built with [Reactor](https://projectreactor.io/), [Netty](https://netty.io/), and a focus on flexibility, Discord4J provides an efficient, non-blocking interface for creating Discord bots.
 
-## Modules
-Discord4J is split up into a few "modules", these allow for extreme customizability and flexibility in how you use the library!
-
-### [Common](https://github.com/Discord4J/Discord4J/tree/v3/common)
-The `common` module contains some simple classes used by most other modules in the library.
-
-### [Rest](https://github.com/Discord4J/Discord4J/tree/v3/rest)
-The `rest` module allows for low level interaction with the [REST portion of the Discord API](https://discordapp.com/developers/docs/reference).
-
-### [Gateway](https://github.com/Discord4J/Discord4J/tree/v3/gateway)
-The `gateway` module allows for low level interaction with the [gateway portion of the Discord API](https://discordapp.com/developers/docs/topics/gateway)
-
-### [Store](https://github.com/Discord4J/Discord4J/tree/v3/store)
-The `store` module provides a set of common interfaces to create efficient caching systems for use in the [core module](https://github.com/Discord4J/Discord4J/tree/v3/core).
-
-### [Core](https://github.com/Discord4J/Discord4J/tree/v3/core)
-The `core` module exposes a high level API for interacting with all facets of the Discord API built on top of all the other modules, this is generally what you'd want to use.
-
-[The API is also available in a few other languages.](https://discordapi.com/unofficial/libs.html)
-
-## Adding Discord4J-Core as a dependency for a project
-Given that `@VERSION@` = the version of Discord4J (this can either be a release version, the (short or long) commit hash or `<git branch>-SNAPSHOT`).
-### With Maven
-In your `pom.xml` add (without the ellipses):
-```xml
-<repositories>
-  <repository>
-    <id>jitpack.io</id>
-    <url>https://jitpack.io</url>
-  </repository>
-</repositories>
-
-<dependencies>
-  <dependency>
-    <groupId>com.github.Discord4J.Discord4J</groupId>
-    <artifactId>Discord4J-core</artifactId>
-    <version>@VERSION@</version>
-  </dependency>
-</dependencies>
-```
-### With Gradle
-In your `build.gradle` add (without the ellipses): 
+## Installation
```

Comment:
Include maven and sbt as well, there's no reason not to

Reply:
The reason is bloat. At most, I think adding maven would be ok. No one is using core in Scala/with SBT.

Reply:
Maven is a must I'd say, and sbt is debatable I guess but it would be consistent with the website and it doesn't really add too much lines.